__tests__/kata1.fizzBuzz.test.js

```diff
@@ -1,19 +1,22 @@
-const { fizzBuzz } = require('../src');
+const fizzBuzz = require('../src/kata1.fizzBuzz.js')
 
 describe('fizzBuzz', () => {
   it('returns Fizz when passed a multiple of 3', () => {
-
+    expect(fizzBuzz(3)).toBe('Fizz')
   });
 
```

Comment:
It might be a good idea to add a few more expect lines to these tests. You could check other multiples of 3 and 5.